std/datetime.d

```diff
@@ -31663,7 +31663,7 @@ SysTime parseRFC822DateTime(R)(R value) @safe
     }
 
     // day-of-week
-    if(std.ascii.isAlpha(value[0]))
+    if(isAlpha(value[0]))
```

Comment:
Honestly, no :) `std.datetime` has over 30k lines, and most of them are not this way. If you want to do the reformatting, I suggest we do it all at once, and only to `std.datetime` as a separate PR.